openlibrary/plugins/upstream/forms.py

```diff
@@ -58,7 +58,10 @@ def find_ia_account(email=None):
     r"^[A-Za-z0-9\-_]{3,20}$", _('Must be between 3 and 20 letters and numbers')
 )
 vpass = RegexpValidator(r".{3,20}", _('Must be between 3 and 20 characters'))
-vemail = RegexpValidator(r".*@.*", _("Must be a valid email address"))
+vemail = RegexpValidator(
+    r"^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,}$",
```

Comment:
In the interest of limiting the scope of this change, let's use the `.*@.*\..*` we were using before ; the current regex would error for e.g. french characters!